shared/shot.js

```diff
@@ -203,7 +203,7 @@ class AbstractShot {
     this.thumbnail = attrs.thumbnail || null;
     this.abTests = attrs.abTests || null;
     this.firefoxChannel = attrs.firefoxChannel || null;
-    this.isOwner = attrs.isOwner === undefined ? undefined : !!attrs.isOwner;
+    this._isOwner = attrs.isOwner === undefined ? undefined : !!attrs.isOwner;
```

Comment:
Since the `isOwner` setter has the undefined check removed, I don't think we need to assign to `_isOwner` directly